src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -104,6 +105,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        model.updateViewedPersonList(new SamePersonPredicate(editedPerson));
         return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson));
```

Comment:
I think it is better to remove String.format here?

Reply:
Line changed to `return new CommandResult(editedPerson.toString());`